src/main/java/com/google/sps/data/OrderHandler.java

```diff
@@ -0,0 +1,135 @@
+// Copyright 2019 Google LLC
```

Comment:
Please move OrderHandler and everything it touches into a separate PR. I have no idea why this needed to be done here.

But this PR is already quite a beast, and adding more here feels like looking for trouble.

Reply:
Done.